lib/benchee/benchmark/measure/time.ex

```diff
@@ -1,11 +1,23 @@
 defmodule Benchee.Benchmark.Measure.Time do
   @moduledoc """
-  Measure the time consumed by a executing function.
+  Measure the time elapsed while executing a given function.
```

Comment:
Should this be private API as well?

Reply:
Here I'm... a bit torn. It's nothing I expect 99% users of benchee to ever need - so probably not. On the other hand, it can contain documentation on how benchee conducts its measurements which might be interesting to the more inclined reader. Also as an example if someone wants to implement their own measurer

Reply:
We don't allow for people to implement their own measurer yet - is that something you see coming in the future? That sort of strikes me as a thing we might not want to let people do, since you've put a _lot_ of time and effort to make sure measurements are taken in the best way possible.

And we can still document in line with `#`s to show how it's done if folks are curious. It's a little tricky since José has said that the _only_ way a library can mark something as private API is to not document it, so we're sort of without options here. My preference is usually to make as much private as possible, and then later on make things public if a good reason comes up to do so.

Reply:
You're right, thanks Devon. Will change it!